sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/ContactDirectoryPage.java

```diff
@@ -48,6 +48,31 @@
   public static final By GRID_HEADERS = By.xpath("//thead//tr//th");
   public static final String RESULTS_GRID_HEADER = "//div[contains(text(), '%s')]";
   public static final By FIRST_CONTACT_ID_BUTTON = By.cssSelector(".v-grid-row-has-data a[title]");
+  public static final By CONTACT_EPIDEMIOLOGICAL_DATA = By.cssSelector("#tab-contacts-epidata");
+  public static final By CASE_LIKE = By.id("contactOrCaseLike");
+  public static final By NEW_ENTRY_EPIDEMIOLOGICAL_DATA = By.id("actionNewEntry");
+  public static final By TYPE_OF_ACTIVITY_DETAILS = By.id("exposureTypeDetails");
+  public static final By TYPE_OF_GATHERING_DETAILS = By.id("gatheringDetails");
+  public static final By TYPE_OF_PLACE_DETAILS = By.id("typeOfPlaceDetails");
+  public static final By INDOORS_OPTION = By.cssSelector("#indoors .v-select-option");
+  public static final By TYPE_OF_GATHERING_COMBOBOX =
+      By.cssSelector(".v-window #gatheringType div");
+  public static final By LATITUDE_INPUT = By.cssSelector(".v-window input#latitude");
+  public static final By LONGITUDE_INPUT = By.cssSelector(".v-window input#longitude");
+  public static final By LATLONACCURACY_INPUT = By.cssSelector(".v-window input#latLonAccuracy");
+  public static final By FACILITY_CATEGORY_COMBOBOX = By.cssSelector(".v-window #typeGroup div");
+  public static final By FACILITY_TYPE_COMBOBOX = By.cssSelector(".v-window #facilityType div");
+  public static final By FACILITY_DETAILS_COMBOBOX = By.cssSelector(".v-window #facility div");
+  public static final By FACILITY_NAME_AND_DESCRIPTION =
+      By.cssSelector(".v-window input#facilityDetails");
+  public static final By CONTACT_PERSON_FIRST_NAME =
+      By.cssSelector(".v-window input#contactPersonFirstName");
+  public static final By CONTACT_PERSON_LAST_NAME =
+      By.cssSelector(".v-window input#contactPersonLastName");
+  public static final By CONTACT_PERSON_PHONE_NUMBER =
+      By.cssSelector(".v-window input#contactPersonPhone");
+  public static final By CONTACT_PERSON_EMAIL_ADRESS =
+      By.cssSelector(".v-window input#contactPersonEmail");
   public static final By EPIDEMIOLOGICAL_DATA_TAB = By.cssSelector("#tab-contacts-epidata");
```

Comment:
please review all locators and rename them to refelct the WebElement type. So the ending should reflect the item in UI (eg: button, input, tab etc)